plugin.video.tweakers/addon.xml

```diff
@@ -27,5 +29,6 @@
     <website>http://www.tweakers.net</website>
     <email></email>
     <source>https://github.com/skipmodea1/plugin.video.tweakers</source>
+    <python3>true</python3>
```

Comment:
Have we decided on this format yet? Feels a bit premature to include?

Reply:
Nothing decided yet

Reply:
Right, i'll close the PR and wait for a decision.

Reply:
why not update the addon without this?